integreat_cms/cms/models/pages/page_translation.py

```diff
@@ -26,45 +17,12 @@ class PageTranslation(AbstractBasePageTranslation):
     Data model representing a page translation
     """
 
-    title = models.CharField(
-        max_length=1024,
-        verbose_name=_("title of the page"),
-    )
-    slug = models.SlugField(
-        max_length=1024,
-        allow_unicode=True,
-        verbose_name=_("Page link"),
-        help_text=__(
-            _("String identifier without spaces and special characters."),
-            _("Unique per region and language."),
-            _("Leave blank to generate unique parameter from title."),
-        ),
-    )
     page = models.ForeignKey(
         Page,
         on_delete=models.CASCADE,
         related_name="translations",
         verbose_name=_("page"),
     )
```

Comment:
```suggestion
```
I guess we could even move this to `AbstractBasePageTranslation` and set the `default_related_name` of `PageTranslation` to `translations` instead of `page_translations`, but probably this would require some further adjustments to the code base.

Reply:
I think this comes with too many caveats for now. Let's ignore this for this PR.